binr/rax2/rax2.c

```diff
@@ -118,7 +118,8 @@ static int help() {
 static int rax(char *str, int len, int last) {
 	float f;
 	ut8 *buf;
-	char *p, newline = '\n', out_mode = (flags & 128)? 'I': '0';
+	char *p, out_mode = (flags & 128)? 'I': '0';
+	char newline[2] = "\n\0";
```

Comment:
if i uses char for next line, null char will be printed too and it's make wrong result.
```
$./x/bin/rax2 -l -D "jLIjfQZ5yojbZGTqxgqpY0VROWQ=" | ./x/bin/rax2 -S
8cb2237d0679ca88db6464eac60aa9634551396400 => 00 is wrong
```
but in string \0 will not printed.
after using string :
```
$./x/bin/rax2 -l -D "jLIjfQZ5yojbZGTqxgqpY0VROWQ=" | ./x/bin/rax2 -S
8cb2237d0679ca88db6464eac60aa96345513964
```